packages/components/src/custom-select-control-v2/test/index.tsx

```diff
@@ -0,0 +1,176 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+/**
+ * Internal dependencies
+ */
+import { CustomSelect, CustomSelectItem } from '..';
+
+describe( 'CustomSelect', () => {
```

Comment:
We also updated `CustomSelectControl` to do that. 😄 I had figured this would happen when merging into the PR where the legacy tests are added (https://github.com/WordPress/gutenberg/pull/57902). But I can update it here so it's a cleaner merge.